atom/browser/atom_browser_client.cc

```diff
@@ -472,6 +481,33 @@ void AtomBrowserClient::SiteInstanceDeleting(
   }
 }
 
+std::unique_ptr<net::ClientCertStore> AtomBrowserClient::CreateClientCertStore(
```

Comment:
@alexeykuzmin please use the changes from
https://github.com/electron/electron/pull/13245/files#diff-5a6becb7577816fc1d0a6ac6dc66db11